src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -134,7 +134,7 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
           />
           <PromoBoxWithButton
             text={
-              "Turn on 'DailyReports' to receive daily analysis and status updates from Recidiviz and CSG."
+              "Turn on 'Daily Reports' to receive briefings based on the data in this scenario, prepared by Recidiviz and CSG."
```

Comment:
@daschi just a note that this copy is being updated and merged shortly, so you will want to make sure it's up-to-date in #165, as well. Thanks!